rust/src/write.rs

```diff
@@ -175,6 +205,87 @@ impl WriteOptions {
         self
     }
 
+    /// Specifies whether the writer should output statistics automatically.
```

Comment:
I don't understand the use of "automatically" - there's not a manual alternative. Maybe:
```suggestion
    /// Specifies whether the writer should output a statistics record in the summary section.
```

Reply:
Fixed in updated docs.